.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,29 @@
+name: Automated javascript build CI
+on:
+  push:
+    paths: 'webapp-admin/**'
```

Comment:
This will trigger only on changes in `webapp-admin` and all the sub-directories of this